### PR TITLE
make data and job name consistent with web UI

### PR DESCRIPTION
**Pull request by houqp**

### CODE REVIEW

File: floyd/cli/data.py [hunk 1]
```diff
@@ -159,18 +161,30 @@ def delete(ids, yes):
             failures = True
             continue
 
-        if not yes and not click.confirm("Delete Data: {}?".format(data_source.name),
+        data_name = normalize_data_name(data_source.name)
+        suffix = data_name.split('/')[-1]
+        if not suffix.isdigit():
+            failures = True
+            floyd_logger.error('%s is not a dataset, skipped.', id)
```
Comment (narenst) [inline floyd/cli/data.py]: We should probably log this in the else section of the if condition following this.

File: tox.ini [hunk 2]
```diff
@@ -8,3 +8,7 @@ commands =
 deps =
     setuptools>=17.1
 
+[flake8]
```
Comment (narenst) [inline tox.ini]: Test comment. Please ignore.

File: floyd/cli/data.py (hunk 1 above)
Comment (houqp) [inline floyd/cli/data.py]: If the name is not ending with version number, we can already conclude that it's not an dataset. I added it here under the assumption that we will have more variants other than `output` in the future.

Comment (narenst) [inline floyd/cli/data.py]: 👍

**Pull request merged**